engine/table/src/main/java/io/deephaven/engine/table/impl/partitioned/LongConstantColumn.java

```diff
@@ -0,0 +1,184 @@
+package io.deephaven.engine.table.impl.partitioned;
+
+import io.deephaven.chunk.ChunkType;
+import io.deephaven.chunk.WritableChunk;
+import io.deephaven.chunk.attributes.Values;
+import io.deephaven.engine.rowset.RowSequence;
+import io.deephaven.engine.rowset.TrackingRowSet;
+import io.deephaven.engine.table.*;
+import io.deephaven.engine.table.impl.select.Formula;
+import io.deephaven.engine.table.impl.select.SelectColumn;
+import io.deephaven.engine.table.impl.sources.InMemoryColumnSource;
+import io.deephaven.engine.table.impl.sources.SparseArrayColumnSource;
+import io.deephaven.engine.table.impl.sources.ViewColumnSource;
+import io.deephaven.util.type.TypeUtils;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * {@link SelectColumn} implementation to assign a constant {@code long} value.
```

Comment:
Because I was already dabbling in creating custom `SelectColumn` implementations, and because to add a `ColumnSource` I'd have to write yet-another custom "copy but with a tweak" implementation. Expressing it as a `SelectColumn` let's me just use `updateView`, which is easier to get right in terms of correctness. I actually kind of want to build a lot more things this way.